shell/platform/linux/fl_engine.cc

```diff
@@ -24,6 +24,11 @@
 // Unique number associated with platform tasks.
 static constexpr size_t kPlatformTaskRunnerIdentifier = 1;
 
+// Use different device ID for mouse and pan/zoom events, since we can't
+// differentiate the actual device (mouse v.s. trackpad)
+static constexpr int32_t kMousePointerDeviceId = 0;
+static constexpr int32_t kPointerPanZoomDeviceId = 1;
```

Comment:
Nit: Do we have a way to share constants like this via our `common/` directory?  Since these are also used in the Mac PR.

Also, I'm not sure if we already have conventions for what these device ids, but if no one else knows then I think this is fine.